File: app/src/test/java/bj/discogsbrowser/login/OAuthTokenFactory.java
package bj.discogsbrowser.login;

import com.github.scribejava.core.model.OAuth1AccessToken;

/**
 * Created by Josh Laird on 09/05/2017.
 */

public class OAuthTokenFactory
{
    public OAuth1AccessToken getEmptyToken()
    {
        return new OAuth1AccessToken("", "");
    }

    public OAuth1AccessToken getValidToken()
    {
        return new OAuth1AccessToken("bj", "lairy");
    }
}

File: app/src/test/java/bj/vinylbrowser/login/OAuthTokenFactory.kt
package bj.vinylbrowser.login

import com.github.scribejava.core.model.OAuth1AccessToken

/**
 * Created by Josh Laird on 21/05/2017.
 */
object OAuthTokenFactory {
    @JvmStatic fun getEmptyToken(): OAuth1AccessToken {
        return OAuth1AccessToken("", "")
    }

    @JvmStatic fun getValidToken(): OAuth1AccessToken {
        return OAuth1AccessToken("bj", "lairy")
    }
}